packages/bcer-api/app/src/notification/entities/notification.entity.ts

```diff
@@ -0,0 +1,74 @@
+import { Column, CreateDateColumn, Entity, PrimaryGeneratedColumn, UpdateDateColumn } from "typeorm";
+import { NotificationRO } from "../ro/notification.ro";
+
+@Entity('notification')
+export class NotificationEntity {
+    @PrimaryGeneratedColumn('uuid')
+    id: string;
+
+    @Column('varchar', {
+        length: 255,
+        nullable: true
+    })
+    sender: string;
```

Comment:
I think it would be good to make this a `FK` to the `User` entity to make sure we can map it back to the actual user that sent it easily

Reply:
We actually have no user record for data portal logins  :'(